src/components/functional/CreatePaymentRequestPage/CreatePaymentRequestPage.tsx

```diff
@@ -83,13 +86,48 @@ const CreatePaymentRequestPage = ({
     makeToast('success', 'Payment request successfully created.');
   };
 
+  const onSaveUserPaymentDefaults = async (userPaymentDefaults: UserPaymentDefaults) => {
+    const client = new ClientSettingsClient(apiUrl, token);
+    const response = await client.saveUserPaymentDefaults(userPaymentDefaults);
+
+    if (response.error) {
+      makeToast('error', response.error.title);
+      return;
+    }
+  };
+
+  const getDefaultUserPaymentDefaults = (): UserPaymentDefaults => {
```

Comment:
This could be rather just a variable, there's no need for a function. We'd even have this into a `constants` file.